index.html

```diff
@@ -50,5 +50,14 @@
         </ul>
       </nav>
     </header>
+    <main>
+      <section id="banner">
+        <div class="banner__text">
+          <h6>Welcome To Cyborg</h6>
```

Comment:
heading should follow the correct order. H6 should not come before h4. Think HTML page as a word document.

Reply:
Fixed it